### ISSUE TITLE
[BUG] Paper trading mode doesn't work in XEMM

### ISSUE THREAD

**Issue by Juggy1028** [CoinAlpha/hummingbot#1672]
**Describe the bug**
// A clear and concise description of what the bug is.

ok, this is my first bug submission ever, so forgive me if it goes astray. On paper trade mode, cross MM strategy, the status command is not returning accurate information. The sell figure has never changed in over an hour of activating the bot and after over 150 trades.

See log of price change:

> 2020-04-26 15:27:54,007 - 23448 - hummingbot.strategy.cross_exchange_market_making.cross_exchange_market_making - INFO - (XMRBTC) The current limit bid order for 0.0475676 XMR at 0.00799076 BTC is now below the suggested order price at 0.00801375. Going to cancel the old order and create a new one... [clock=2020-04-26 19:27:54+00:00]

See status message:
![image](https://user-images.githubusercontent.com/38084712/80317617-df739f80-87d2-11ea-9c0e-91e4ad225a93.png)

**Steps To Reproduce**
// A concise description of the **steps to reproduce the buggy behavior**:

Run cross MM bot, keep using status command function, notice the sell side price on active trades never changes.

Here are the steps to reproduce the issue (see attachments in the section below):
1. Set up your configuration file with these parameters: …
![image](https://user-images.githubusercontent.com/38084712/80317441-c28a9c80-87d1-11ea-859c-1f07a08cc754.png)

I'm not sure if the screenshot is working so I'm pasting it as well:

```
########################################################
###   Cross exchange market making strategy config   ###
########################################################

template_version: 3
strategy: cross_exchange_market_making

# The following configuations are only required for the
# cross exchange market making strategy

# Exchange and token parameters
maker_market: binance
taker_market: kucoin
maker_market_trading_pair: XMR-BTC
taker_market_trading_pair: XMR-BTC

# Minimum profitability target required to place an order
# Expressed in decimals: 0.01 = 1% target profit
min_profitability: 0.003

# Maximum order size in terms of base currency
order_amount: 2.0

# Have Hummingbot actively adjust/cancel orders if necessary.
# If set to true, outstanding orders are adjusted if
# profitability falls below min_profitability.
# If set to false, outstanding orders are adjusted if
# profitability falls below cancel_order_threshold.
active_order_canceling: false

# If active_order_canceling = false, this is the profitability/
# loss threshold at which to cancel the order.
# Expressed in decimals: 1 = 1% target profit
cancel_order_threshold: 5.0

# An amount in seconds, which is the minimum duration for any
# placed limit orders. Default value = 130 seconds.
limit_order_min_expiration: 60.0

# If enabled (parameter set to `True`), the strategy will place the order on top of the top bid and ask,
# if its more profitable to place it there
adjust_order_enabled: true

# An amount expressed in base currency which is used for getting the top bid and ask,
# ignoring dust orders on top of the order book
top_depth_tolerance: 0.0

# An amount in seconds, which is the minimum amount of time interval between adjusting limit order prices
anti_hysteresis_duration: 30.0

# An amount expressed in decimals (i.e. input of `1` corresponds to 1%), which is the maximum size limit of new limit orders,
# in terms of ratio of hedge-able volume on taker side.
order_size_taker_volume_factor: 1.0

# An amount expressed in decimals (i.e. input of `1` corresponds to 1%), which is the maximum size limit of new limit orders,
# in terms of ratio of asset balance available for hedging trade on taker side
order_size_taker_balance_factor: 0.995

# An amount expressed in decimals (i.e. input of `1` corresponds to 1%), which is the maximum size limit of new limit orders,
# in terms of ratio of total portfolio value on both maker and taker markets
order_size_portfolio_ratio_limit: 0.1667

``

2. Start the bot using this command: ...
start.. What else?

**Screenshots**
// If applicable, add screenshots to help explain your problem.

![image](https://user-images.githubusercontent.com/38084712/80317416-af77cc80-87d1-11ea-992a-44fdc768f452.png)


**Release version**
// Include your bot's version number (Can be found at the upper left corner of your CLI).

**Attachments**
// Please attach your config file and log file to help us debug. It would be difficult for us to help you without those! 


*WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!*

**Optional: your discord username:**

**Comment by Juggy1028** [comment]
Sorry, bot version 0.26.0
[logs_conf_xemm_4.log](https://github.com/CoinAlpha/hummingbot/files/45

36303/logs_conf_xemm_4.log)

**Comment by devoger** [comment]
Tested on docker build and source build, same problem. The bot keeps using the same ask /bid price after several hours of trading.
So far the problem occurs between Binance and Kucoin.

![Screen Shot 2020-04-27 at 14 19 38](https://user-images.githubusercontent.com/8411294/80339997-63d92780-8892-11ea-8c90-ebbcea89d8c6.png)

**Comment by mifeng** [comment]
I think this issue is related to https://github.com/CoinAlpha/hummingbot/issues/1666. We will schedule work on this issue.

**Comment by vic-en** [comment]
Now works:
**Before**
<img width="970" alt="before" src="https://user-images.githubusercontent.com/31972210/83201542-99955880-a13d-11ea-960d-7fbbb26f0c7f.png">

**After**
<img width="956" alt="after" src="https://user-images.githubusercontent.com/31972210/83201559-aa45ce80-a13d-11ea-9644-247685a7646a.png">


**History**
Ask /bid price orders changed accordingly throughout a period of 52 minutes.
<img width="954" alt="history" src="https://user-images.githubusercontent.com/31972210/83201691-f8f36880-a13d-11ea-8714-62ca77759081.png">

**Comment by mifeng** [comment]
This bug was fixed by revisions made in v0.27.0
